app/src/main/java/dev/arkbuilders/arkmemo/models/Tag.kt

```diff
@@ -0,0 +1,7 @@
+package dev.arkbuilders.arkmemo.models
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class Tag(val value: String) : Parcelable
```

Comment:
This is not yet implemented in MVP version, but might be there in post MVP.
So I'd like to keep it around.